styleguide/Views/styleguide.blade.php

```diff
@@ -9,11 +9,11 @@
         <h2>Two column example</h2>
 
         <div class="row -mx-4 flex flex-wrap">
-            <div class="w-full sm:w-1/2 px-4">
+            <div class="w-full sm:w-1/2 px-4 content">
```

Comment:
Looks like this whole page is already wrapped in .content so we shouldnt need it here

```suggestion
            <div class="w-full sm:w-1/2 px-4">
```